src/Core/Checkout/Customer/SalesChannel/LoadWishlistRouteResponse.php

```diff
@@ -9,28 +9,22 @@
 use Shopware\Core\Framework\Struct\ArrayStruct;
 use Shopware\Core\System\SalesChannel\StoreApiResponse;
 
+/**
+ * @extends StoreApiResponse<ArrayStruct<string, CustomerWishlistEntity|EntitySearchResult<ProductCollection>>>
```

Comment:
Can we use object shapes with the array struct?

ArrayStruct{'wishlist': CustomerWishlistEntity, 'products': SearchResult}

Reply:
ok not possible on a class

Reply:
I found a way :grin: